pkg/webservice/handlers.go

```diff
@@ -521,3 +523,24 @@ func updateConfiguration(conf string) (string, error) {
 	}
 	return "", fmt.Errorf("config plugin not found")
 }
+
+func getPartitionQueues(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	writeHeaders(w)
+	_, partitionExists := vars["partition"]
+	if len(vars) == 0 || !partitionExists {
+		http.Error(w, "Mandatory parameters are missing in URL path. Please check the usage documentation", http.StatusBadRequest)
+		return
+	}
+	var partitionQueuesDAOInfo dao.PartitionQueueDAOInfo
+	var partition = schedulerContext.GetPartition(vars["partition"])
+	if partition != nil {
+		partitionQueuesDAOInfo = partition.GetPartitionQueues()
+	} else {
+		http.Error(w, "partition not found", http.StatusInternalServerError)
```

Comment:
Wouldn't be better to return with BasRequest error in this case as well? If the partition is not found that means that the user entered a wrong partition name.

Reply:
Taken care.